### ISSUE TITLE
Disable D3D by default.

### ISSUE THREAD

**Issue by FullBleed** [RPTools/maptool#2982]
**Is your feature request related to a problem? Please describe.**
Display issues caused by d3d=true in MT is probably the second biggest help issue we see on Discord (second to port forwarding) and the manual editing of the CFG file to fix the problem still leads to a lot of fails.

**Describe the solution you'd like**
Disable d3d by default in  the cfg:
java-options=-Dsun.java2d.d3d=false

**Describe alternatives you've considered**
Java currently won't let users do it within MT.  Manually editing still provides a techinical hurdle for many users.  And while a standalone app to edit the cfg is listed as a git issue, that is a taller order than just changing the default.  Not to mention...  

**Additional context**
In my (albeit minor) testing, the performance of D3D=true was either minimally positive or minimally negative, **with the negatives being more prominent**.  So not only is the current default annoying many users and making MT look bad, it actually might be a net negative.  And, as yet, I don't think we have any reports of FALSE causing issues.

**Comment by CombatWombatXL** [comment]
I would like to add support behind this change request. I frequently run into issues where this tends to be the fix. I prefer map tools and have had to essentially remote into my tech illiterate friends' computers to fix this issue for them on multiple occasions. Having this as a default would be a huge help.

**Comment by FullBleed** [comment]
Yeah, this issue has been around for years... and I'm not sure if we're seeing it more because of newer graphic drivers, newer graphic cards, the newer java versions MT uses, or because people just can't easily fix the problem themselves due to MT's "broken" ability to disable D3D within the app itself anymore.  Whatever the reason, it has become an unnecessary inconvenience for many MT users.
